### ISSUE TITLE
Initial upload of code

### ISSUE THREAD

**Issue by jatkinson1000** [GO-SHIP-Oceanography/plot-bottles-over-topography#1]
Hi @ShuwenTan-PO here is your repository for the CPD bottle code!

I have created it with an empty README.md, a .gitignore for python code, and an MIT License.
I have invited you to be an administrator for this repository along with @lcimoli.

The first task is an initial upload of the code.
We suggest you do this in a branch so that we can review and tidy things before adding them to main.
Below are instructions for this copied from the [Contribution Guidelines](https://github.com/GO-SHIP-Oceanography/.github/blob/main/contributing.md#adding-code):

Navigate into the folder that contains your code and then run, from the terminal:
```
git init
git remote add origin git@github.com:GO-SHIP-Oceanography/plot-bottles-over-topography.git
git pull origin main
```
to pull any base documents (e.g. license).  

_**Note**: This assumes you are accessing a repository via ssh (advised). If you prefer to
access via https you can use
`git remote add origin https://github.com/GO-SHIP-Oceanography/plot-bottles-over-topography.git`
and will be prompted for your GitHub username and password each time you access._

We suggest that you perform your initial upload from a branch, allowing the code to be
reviewed before it is contributed to the main branch, and checked to ensure that it
works and is reproducible.
To do this run:
```
git checkout -b first-commit
```

Following this you can add files using [`git add`](https://github.com/git-guides/git-add),
commit them using [`git commit`](https://github.com/git-guides/git-commit) and push them
to the repository using [`git push`](https://github.com/git-guides/git-push) as usual.
The [Turing Way guide](https://the-turing-way.netlify.app/reproducible-research/vcs/vcs-git)
is a useful guide for those new to these ideas.

Once you are ready for the initial upload run:
```
git push origin first-commit
```
to upload the code to GitHub.

You can then open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
from the repository webpage on GitHub to ask to add your code to the main branch.

If you are unsure about any part of this process we are happy to guide you through it.

**Comment by jatkinson1000** [comment]
Hi @ShuwenTan-PO I'm back around next week if you wanted any help with this 😃

**Comment by ShuwenTan-PO** [comment]
Hi @jatkinson1000, sorry for the delay. I finally managed to update the code to the new branch. It's the older version of the code and @lcimoli has an updated one. So maybe we can get the code irritating from here. I was able to follow most of the instructions, except for some struggles with setting up the SSH (my public key was not associated with my GitHub account) and some steps before the initial push (I forgot that I needed to [git add](https://github.com/git-guides/git-add) and [git commit](https://github.com/git-guides/git-commit) code before [git push](https://github.com/git-guides/git-push)). Maybe we can provide some info in the instruction: 
- we can add "make sure that your SSH keys ([check your existing keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys) or [create new keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)) are added to your GitHub account." to the ```git remote add origin``` session.
- we can provide an example [git add](https://github.com/git-guides/git-add) and [git commit](https://github.com/git-guides/git-commit) code before the [git push](https://github.com/git-guides/git-push).
```
git add .
git commit -m "first commit"
```
Does that make sense?

**Comment by jatkinson1000** [comment]
Hi @ShuwenTan-PO this is great, thank you.

And thanks for raising those points - that's really useful feedback and I'll make sure we update the guidance accordingly for future users.

If it is OK with you I will clean up a couple of things with your original push (e.g. remove the .DS_Store file) and then conduct a friendly code review.
If that sounds OK let me know and I'll make a start, keeping you updated.

**Comment by ShuwenTan-PO** [comment]
@jatkinson1000 of course! Feel free to clean up the code, I didn't notice that! Thank you!!!
